xbmc/pvr/PVRDatabase.cpp

```diff
@@ -548,8 +582,9 @@ int CPVRDatabase::GetClientID(const std::string& addonID, unsigned int instanceI
     return std::atoi(idString.c_str());
 
   // Create a new entry with a new client id in clients table.
-  sql = PrepareSQL("INSERT INTO clients (iPriority, sAddonID, iInstanceID) VALUES (%i, '%s', %i)",
-                   0, addonID.c_str(), instanceID);
+  sql = PrepareSQL("INSERT INTO clients (iPriority, sAddonID, iInstanceID, "
+                   "sDateTimeFirstChannelsAdded) VALUES (%i, '%s', %i, '%s')",
+                   0, addonID.c_str(), instanceID, "");
```

Comment:
Why the empty sting here? Is this because we couldn't have had this value in this version of the DB? If so, maube add a comment for future readers?

Reply:
Thanks, will add a comment. Reason is that we don't have that information at that point in time. It will be added later, on-demand.